### ISSUE TITLE
[tests] timeout on appveyor

### ISSUE THREAD

**Issue by svenefftinge** [theia-ide/theia#860]
Tests on appveyor fail occasionally:

```
lerna ERR! execute   1) NodeFileSystem 04 #move Moving a non-empty directory to an empty directory. Source folder and its content should be moved to the target location.:
lerna ERR! execute      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

**Comment by epatpol** [comment]
Should we find a way to increase the timeout only for tests on Windows? Or maybe we could increase it globally...

**Comment by epatpol** [comment]
Or if it's always the same test we could increase the timeout for this one only (I haven't checked if it's always the same culprits).

**Comment by svenefftinge** [comment]
I think just increasing the timeout a bit is fine. If it hangs it will fail anyhow.

**Comment by kittaakos** [comment]
Does this timeout adjustment apply only on the FS tests?

**Comment by hexa00** [comment]
Fixed in #980
